actions/health-and-version-check/src/health-check.ts

```diff
@@ -4,25 +4,27 @@ import axios from 'axios';
 export type VersionResult = { status: boolean, response: string }
 
 export async function CheckWebAppHealth(
-    webAppName: string, 
-    healthUri: string,
-    numberOfSeconds?: number): Promise<boolean> {
-        if (!numberOfSeconds) numberOfSeconds = 300;
-
-        const url = `https://${webAppName}.azurewebsites.net${healthUri}`;
-        await new SleepTimer().sleep(10000);
-        return await checkHealth(url, webAppName, numberOfSeconds);
+    webAppName: string,
+    healthUrl: string,
+    numberOfSeconds?: number
+): Promise<boolean> {
+    if (!numberOfSeconds) numberOfSeconds = 300;
+    await new SleepTimer().sleep(10000);
+    return await checkHealth(healthUrl, webAppName, numberOfSeconds);
 }
 
-export async function CompareVersionStrings(webAppName: string, versionUri: string, expectedVersionString: string): Promise<VersionResult> {
+export async function CheckVersion(
+    versionUrl: string,
+    expectedVersionString: string
+): Promise<VersionResult> {
+    info(`Checking version at: ${versionUrl}`);
+    info(`Searching for: ${expectedVersionString}`)
     let result: VersionResult = { status: false, response: '' };
-    const url = `https://${webAppName}.azurewebsites.net/${versionUri}`
+    const response = await axios.get(versionUrl);
```

Comment:
the `versionUrl` is only the endpoint so we still need to construct the string 

```suggestion
    const response = await axios.get(`https://${webAppName}.azurewebsites.net/${versionUri}`);
```

Reply:
The parameter name's description isn't the best. It should probably be more like `versionEndPoint` or something. *scaffolding my past self for bad naming practices 😒